maintainers/maintainer-list.nix

```diff
@@ -4186,7 +4186,7 @@
     matrix = "@clot27:matrix.org";
   };
   cloudripper = {
-    email = "dev+nixpkgs@cldrpr.com";
+    email = "other.wing8806@fastmail.com";
```

Comment:
Why is this part of the diff? Bad rebase/merge?

Reply:
Bad Merge. Working on it properly today, apologies was a bit haphazard yesterday.

Reply:
fixed now.